README.md

```diff
@@ -1505,6 +1479,14 @@ condition](#safe-assignment-in-condition).
     tmp = a * 7
     tmp * b / 50
   end
+  
+  #stabby required if there are default arguments to the lambda
```

Comment:
There should be a space after the `#` according to the [comments styleguide](https://github.com/goldstar/ruby-style-guide#hash-space).